src/odemis/driver/technolution.py

```diff
@@ -1402,6 +1418,18 @@ def _setCellTranslation(self, cellTranslation):
                                      "Cell translation parameter values remain unchanged." %
                                      (row, column))
 
+        # check: max cell translation <= cell complete size - effective cell size = max possible cell translation
+        max_cell_translation = numpy.max(numpy.array(cellTranslation), axis=(0, 1))  # (max_x, max_y)
+        eff_cell_size = numpy.array(numpy.array(self.parent._ebeam_scanner.resolution.value) /
+                                   numpy.array(self.shape[0:2]), dtype=int)
```

Comment:
Why isn't this done the same way as in setResolution()? It'd be easier to read if it was exactly the same code. I think that the code `numpy.array(resolution) / self.parent._mppc.shape[0:2]` would work fine here. If you really want an int, you could use `//`.